community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/store/BatchingNeoStores.java

```diff
@@ -220,41 +224,22 @@ public static void createStore( FileSystemAbstraction fileSystem, String storeDi
 
     private NeoStores newNeoStores()
     {
-        return newNeoStores( storeDir );
-    }
-
-    private NeoStores newNeoStores( File storeDir )
-    {
-        StoreFactory storeFactory = newStoreFactory( pageCache, recordFormats, storeDir );
-        return storeFactory.openAllNeoStores( true );
+        return newStoreFactory( DEFAULT_NAME, DELETE_ON_CLOSE ).openAllNeoStores( true );
```

Comment:
Heh yeah, removing `DELETE_ON_CLOSE`. Wait... why did all the tests go green, I wonder